projects/1.18/assets/macaws-roofs-oh-the-biomes-youll-go/macawsroofsbyg/lang/zh_cn.json

```
@@ -0,0 +1,452 @@
+{
+	"itemGroup.tab_mrbyg": "Macaw的屋顶-你将去的生物群系附属",

```

```suggestion
	"itemGroup.tab_mrbyg": "Macaw的屋顶丨BYG附属",
```
BYG BOP这些可以取缩写也可以全称（个人意见）